### PR TITLE
Pagination magic button in collection view, profile and spotlight

### LINKED ISSUE #1058: Add magic random where we have pagination component
This is continued from #1044  #1036

We should add it on 
- collection view, [test on Talisman](https://deploy-preview-1044--nftkodadot.netlify.app/rmrk/collection/b6e98494bff52d3b1e-SPIRIT?page=6)
- artist profile, test on [Zeitgeist Tarot Minter](https://deploy-preview-1044--nftkodadot.netlify.app/rmrk/u/CwXa36Dddsm4h7T1LDnXvtdTFmSfUg7ZVygZ7mv8m6DE19s?page=8) 
- spotlight 

@prachi00 delivered previously, may she want to do this as well?

### PR DESCRIPTION
**Pull request by prachi00**
**Thank you for your contribution** to the KodaDot NFT gallery.

👇 Let's do a quick check before the merge. 

### PR type
- [ ] Bugfix 
- [ ] Feature 
- [x] Refactoring 

### Before submitting Pull Request, please make sure:
- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted screenshot of demonstrated change in this PR

### Optional
- [x] I've tested PR on mobile and everything works
- [ ] I found edge cases

### What's new? 
- [x] PR closes #1058
- [x] Added magic button in spotlight, collection view and artist profile 

### Had issue bounty label ?
- [x] Fill up your KSM address: [Payout](
https://kodadot.xyz/transfer/?target=EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU
)

### Community participation
- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot
- [x] My fix has changed **something** on UI, a screenshot for others, is best to understand changes.
Screenshot 2021-11-08 at 5 19 55 PM

### CODE REVIEW

File: src/components/rmrk/Gallery/CollectionItem.vue [hunk 1]
```diff
@@ -54,7 +54,7 @@
 
       <Search v-bind.sync="searchQuery">
         <b-field>
-          <Pagination simple replace preserveScroll :total="total" v-model="currentValue" :per-page="first" />
+          <Pagination className="mb-0" hasMagicBtn simple replace preserveScroll :total="total" v-model="currentValue" :per-page="first" />
```
Comment (roiLeo) [inline src/components/rmrk/Gallery/CollectionItem.vue:57]: you don't need to create a className @prop as ```class=""``` will work

**Review by roiLeo — CHANGES_REQUESTED** [review]
It seems that I can't reach last page by clicking random btn
Tested on: /rmrk/u/FSJkXLHuWYfB8DYejjFKX4bkFjb7dMt9RvVpq6D5LUQLY6x

I still got a glitch on profile/collections
Screenshot 2021-11-08 at 13-49-36 NFT Artist Profile on KodaDot

File: src/components/rmrk/Gallery/CollectionItem.vue (hunk 1 above)
Comment (prachi00) [reply src/components/rmrk/Gallery/CollectionItem.vue:57]: @roiLeo it throws an error though that you cant use keywords as props

**Comment by prachi00** [comment]
> It seems that I can't reach last page by clicking random btn Tested on: /rmrk/u/FSJkXLHuWYfB8DYejjFKX4bkFjb7dMt9RvVpq6D5LUQLY6x
> 
> I still got a (spacing) glitch on profile/collections Screenshot 2021-11-08 at 13-49-36 NFT Artist Profile on KodaDot

@roiLeo These are fixed

File: src/components/spotlight/SpotlightTable.vue [hunk 2]
```diff
@@ -181,7 +192,34 @@ export default class SpotlightTable extends Mixins(TransactionMixin) {
...
+ }
+ @media only screen and (max-width: 768px) {
+  .magicBtn {
+    top: 4rem;
+    position: relative;
+  }
+   .level-right {
+    margin-left: 2rem;
+    margin-right: 0rem;
+  }
```
Comment (roiLeo) [inline src/components/spotlight/SpotlightTable.vue:223]:
```suggestion
  .magicBtn {
    top: 4rem;
    position: relative;
  }
  .level-right {
    margin-left: 2rem;
    margin-right: 0rem;
  }
```

**Review by roiLeo — APPROVED** [review]
LGTM

**Comment by yangwao** [comment]
Hey @prachi00  Found this At https://deploy-preview-1064--nftkodadot.netlify.app/rmrk/u/CwXa36Dddsm4h7T1LDnXvtdTFmSfUg7ZVygZ7mv8m6DE19s?page=109

Probably missing spacing 
image

Otherwise it's good to merge

**Comment by prachi00** [comment]
> Hey @prachi00 Found this At https://deploy-preview-1064--nftkodadot.netlify.app/rmrk/u/CwXa36Dddsm4h7T1LDnXvtdTFmSfUg7ZVygZ7mv8m6DE19s?page=109
> 
> Probably missing spacing image
> 
> Otherwise it's good to merge
Screenshot 2021-11-09 at 4 07 51 PM


I see it like this tho, not reproducible for me

**Comment by yangwao** [comment]
Interesting doesn't happen to me either, I found some quirk probably. LGTM then
Sent payout https://kusama.subscan.io/extrinsic/0x8253c18dd882b9783b4778e1ef90b1c82b091505ea426269bbbe530c7ac08aa3

**Pull request merged**